input.go

```diff
@@ -1017,3 +1017,83 @@ type UsersFilterInput struct {
 	Arg  *string         `json:"arg,omitempty" yaml:"arg,omitempty" example:"example_arg"`      // Value to be filtered. (Optional.)
 	Type *BasicTypeEnum  `json:"type,omitempty" yaml:"type,omitempty" example:"does_not_equal"` // The operation applied to value on the field. (Optional.)
 }
+
+// create a new AlertSourceServiceDeleteInput by passing an ID string
+func NewAlertSourceServiceDeleteInput(value string) *AlertSourceServiceDeleteInput {
```

Comment:
Theres alot of lines of code in this file that are not covered by tests?  What the purpose of adding these uncovered lines?

Reply:
@rocktavious originally the concern was that we would need these factory methods so that we could create these inputs using just a `string`. However that is not necessary so I'm reverting this change to input.go, as well as removing the code that generated this since it's also not necessary.

I do think we'll re-add something like this down the road when we start generating more opslevel-go code outside of just input.